.jenkins

```diff
@@ -2,7 +2,12 @@
 
 pipeline {
 
-  agent any
+  agent {
+    label 'golang-next'
+  }
+  environment {
+    GITHUB_TOKEN = credentials('vio-bot-gh-token')
+  }
```

Comment:
also, since we define the `GITHUB_TOKEN` here, do we still need it defined in the Tagged Release stage?

Reply:
nope :)